cmp/testdata/diffs

```diff
@@ -1046,6 +1046,26 @@
 + 	},
   }
 >>> TestDiff/Reporter/LargeStandaloneString
+<<< TestDiff/Reporter/MostlyTextString
```

Comment:
This output is suboptimal due to #244.

Ideally it would look like:
```
  strings.Join({
  	"org-4747474747474747,bucket-4242424242424242:m,tag1=a,tag2=aa",
- 	",\xff=_value",
  	" _value=2 11\norg-4747474747474747,bucket-4242424242424242:m,tag1=a,tag2=bb",
- 	",\xff=_value",
  	" _value=2 21\norg-4747474747474747,bucket-4242424242424242:m,tag1=b,tag2=cc",
- 	",\xff=_value",
  	" _value=1 21\norg-4747474747474747,bucket-4242424242424242:m,tag1",
  	"=a,tag2",
- 	",\xff=_value",
  	"=dd _value=3 31\norg-4747474747474747,bucket-4242424242424242:m,tag1",
- 	",\xff=_value",
  	"=c _value=4 41\n",
  }, "")
```

Reply:
all the same, this is _way_ better, thanks!